coalib/parsing/DefaultArgParser.py

```diff
@@ -72,6 +72,12 @@ def default_arg_parser(formatter_class=None):
                             help='Files that will be analyzed will be '
                                  'restricted to those in the globs listed '
                                  'in this argument as well the files setting')
+    arg_parser.add_argument('--caching',
```

Comment:
I think we need a arg to clear this cache also ?
What if the user wants coala to just flush the cache and save a new cache ?
In such a case, *every* time I run with `--caching` it uses the old cache, and I can never tell it to just flush and redo it's caching (Because I think something is wrong for example?) Or maybe we can make this into something like `--caching=flush` ?